### DISCUSSION TITLE
AWS Managed Prometheus with Coroot via Helm?

### DISCUSSION THREAD

**Discussion by sarasensible** [coroot/coroot#214] [Q&A]
Is it possible to install Coroot via Helm with AWS Managed Prometheus?  Looking at the Helm charts there is a bias for self-hosted Prometheus - doesn't seem to be possible to deploy coroot with an external Prometheus url.

Thanks!

**Comment by def** [comment]
Coroot can leverage an external Prometheus, and you can configure this using the [existingPrometheus](https://github.com/coroot/helm-charts/blob/main/charts/coroot/values.yaml#L393) Helm variable. In practice, Coroot uses Prometheus only as temporary metric storage, fetching metrics and storing them on disk for faster access. Therefore, I don't see significant value in using AWS Managed Prometheus.

**Reply by sarasensible** [reply]
Thanks so much for the response!  Our use case is having AWS Managed Prometheus handle the long term storage of metrics since I don't believe coroot does that.

How do I configure the existingPrometheus value to have a url - like a workspace remote read endpoint - with Helm?  I see the following config:

```
  service:
    serviceName: prometheus-stack-1-kube-pr-prometheus
    servicePort: 9090
    serviceNamespace: monitoring
``` 
but how would I adapt this for a url?  Thanks so much!

**Reply by sarasensible** [reply]
There is an additional requirement in order to use AMP - it needs sigv4 auth which Grafana supports.  I don't see a configuration option for this unfortunately.
